documents/WCA Bylaws.md

```diff
@@ -6,7 +6,7 @@
 <div style="margin: 200px auto; font-size: 1.8em;">
 # World Cube Association
 </div>
-<img src="https://www.worldcubeassociation.org/files/WCAlogo_notext.svg" style="width: 60%; margin: auto;">
+<img src="../assets/WCAlogo_notext.svg" style="width: 60%; margin: auto;">
```

Comment:
Actually I seemed to have missed this, what is this change being made to the Bylaws and should it be in this PR?

Reply:
This was part of a technical commit from Philippe. It's just about how the WCA logo is being put into the document. Instead of fetching it over the web, the .svg file is just one folder away now. :)
So this obviously doesn't change anything about the content of the Bylaws.

Reply:
Right, that's exactly what Leon explained!